crypto/libtomcrypt/digit-bit.patch

```diff
@@ -0,0 +1,11 @@
+--- libtomcrypt/src/math/ltm_desc.c	2018-07-01 13:49:01.000000000 -0700
++++ libtomcrypt/src/math/ltm_desc.c.new	2021-06-06 22:42:12.877254274 -0700
```

Comment:
should we upstream both patch to mainline

Reply:
I can upstream the flush, the other is already fixed is the development branch but there has not been a release created yet (there is a ticket discussing it).